main.cpp

```diff
@@ -318,6 +286,31 @@ Part 1c - Step 2: Assignment
         Brightness (double)
         area in cm2 (int)
         brand (std::string)
+
+    Oscillator:
+        pitch (int)
+        octave (int)
+        type (std::string)
+        playSound (bool)
+    LFO:
+        frequency (int)
+        depth (int)
+        type (std::string)
+        isActive (bool)
+    Envelope:
+        Attack (int)
+        Sustain (int)
+        Deacy (int)
+        Release (int)
+    Filter:
+        Resonance (int)
+        Cutoff (int)
+        type (std::string)
+    Keybed:
+        number of keys (int)
+        type of keys (std::string)
+        hasAftertouch (bool)
```

Comment:
Delete this.  I'm not even sure why you wrote this here, as the instructions did not specify to.
- [ ] revise